src/views/Portal/components/ApplicationForm/index.view.tsx

```diff
@@ -198,7 +198,7 @@ const ApplicationForm: React.FC = () => {
         <div className='application-form-component__response'>
           {successOnSubmit === "submitted" ? "submitted" : ""}
         </div>
-        <div className='application-form-component__response'>
+        <div className='application-form-component__response-err'>
           {successOnSubmit === "error submitting" ? "Could Not Submit" : ""}
```

Comment:
Can we update the message to be: "CruzHacks Cloud had an error processing your application. There may be a high bandwidth of users at this moment. Our engineers have been alerted! Try again soon!"

Reply:
Sure! will do